packages/sdk-v2/package.json

```diff
@@ -62,6 +62,7 @@
     "@usedapp/core": "^1.0.9",
     "eventemitter3": "^4.0.7",
     "react-native": "^0.70.2",
+    "react-native-restart": "^0.0.24",
```

Comment:
why do we need restart?

Reply:
i replaced location.reload with cross-platform implementation from GoodDApp
it uses restart on native

Reply:
we could move it to the dev or peer deps